addons/content-index/src/main/java/org/commonjava/indy/content/index/conf/ContentIndexConfig.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2013 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.content.index.conf;
+
+import org.commonjava.indy.conf.IndyConfigInfo;
+import org.commonjava.web.config.annotation.ConfigName;
+import org.commonjava.web.config.annotation.SectionName;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.InputStream;
+
+@SectionName( ContentIndexConfig.SECTION )
+@ApplicationScoped
+public class ContentIndexConfig
+        implements IndyConfigInfo
+{
+    public static final String SECTION = "content-index";
+
+    public static final String AUTH_INDEX_PARAM = "support.authoritative.indexes";
+
+    private boolean authIndex;
```

Comment:
Let's make this a Boolean, use a DEFAULT_AUTHORITATIVE_INDEXES constant with the default value, and use a ternary expression in isAuthIndex() to return the default constant if the value is null.

That will allow us to set the default value to true (without worrying about how the configuration bindings might change field values) someday if we want to.